### DISCUSSION TITLE
[紧急情况突发]新收容物突破收容

### DISCUSSION THREAD

**Discussion by xiaosheng31** [share121/inter-knot#3229] [其他]
紧急情况据最新情况scp-049[又名疫医]在继scp-096突破收容之后紧跟着突破收容其路径与scp-096路线大致吻合都逃向新艾利都方向九尾狐收容二队已经前往收容现049在进入空洞后信号消失因空洞技术探寻不成熟所以我组织特聘一名绳匠协同小队进入空洞进行收容作业
现发布scp-049基本资料
项目编号：SCP-049
项目等级：Euclid
描述：SCP-049是一个人形实体，身高1.9米，外形与中世纪的瘟疫医生相符。虽然SCP-049看上去穿着厚长袍和表明其职业的陶制面具，但这些服装似乎是随着时间的推移从SCP-049的体内生长而出的，现已几乎无法将服装与服装下的体形区分开来。尽管如此，X光检查表明，SCP-049的表层之下确实有人类骨骼结构。SCP-049能够以多种语言发言，但倾向于使用英语或中世纪法语[2](javascript:;)。虽然通常情况下SCP-049亲切友好，愿意與基金会人员合作，但若感觉到它所谓的“瘟疫”之存在便会变得暴躁，甚至极具攻击性。尽管基金会研究人员目前还不清楚这种瘟疫的准确性质，但它似乎是SCP-049极为关心的问题。
SCP-049会对其认为受瘟疫影响的人类个体产生敌意，经常会达到应加以控制的程度。如果不加控制，SCP-049通常会试图杀死该人类个体；SCP-049能够通过直接的皮肤接触使生物体的所有生理机能停止。发生这种情况的原因尚不清楚，对SCP-049受害者的尸检均无定论。SCP-049在杀人后曾表现出沮丧或悔恨，表示这样的行为对消灭“瘟疫”帮助极小，但接下来通常会使用其总是随身携带的一个黑色医疗包中的器具对尸体进行粗糙的手术。手术往往会导致SCP-049-2出现，但并非总能“成功”。
特殊收容措施：SCP-049被收容在Site-19的研究区（Research Sector）-02中的标准安全人形收容单元内。试图运输SCP-049前必须事先予以镇静。在运输过程中，SCP-049必须被固定在Ⅲ级人形限制系带（包括锁圈和延伸限制装置）内，并由至少两名武装守卫监控。
虽然SCP-049通常会与大部分基金会人员合作，但其情绪爆发或行为的突然变化必须通过加强武力应对。任何人员在任何情况下均不得在爆发期间直接接触SCP-049。若SCP-049表现出攻击性，可应用薰衣草（L. multifida）对其的镇静作用予以控制。一旦被镇静，SCP-049通常将变得温顺，并将几乎毫不抵抗地回归收容状态。
为便于持续收容SCP-049，每两周应向该实体提供一具最近死亡的动物（通常是牛或其他大型哺乳动物）尸体供其研究。成为SCP-049-2实体的尸体将被从SCP-049的收容室中移出并焚毁。不再允许SCP-049与人类对象互动，也应拒绝其提供人类对象的请求。
产物;SCP-049手术后几分钟，SCP-049-2将恢复生命体征并重新活跃。然而，SCP-049-2似乎没有更高级的大脑功能，它会漫无目的地漫游，直到遇到活人。此时，SCP-049-2的肾上腺素和内啡肽水平会增加到约300%。它会试图杀死所有能找到的人，然后回到无意识状态并继续徘徊，直到遇到另一个活人.
注:此文件已解密可放心观看
![screenshot-1724090541509](https://github.com/user-attachments/assets/6a40867e-0987-4151-9291-9de4766a4f09)
![screenshot-1724090862187](https://github.com/user-attachments/assets/eec675b4-3676-419c-b9a0-186d702da7a7)
最后看到此消息的无关人员请尽量减少进入空洞作业的时间和次数以及各位盗洞客也请熟知,请勿自误.
在收容作业期间小队也将暂时封闭scp-049所在的空洞直到scp-049和acp-096全部收容完成
如有人在空洞作业遇到scp-049请立即离开并通知收容人员请勿与其缠斗

**Comment by Amzen10086** [comment]
嘿，伙计怎么不早说。我的那队人才进入空洞
